{"id":696325,"title":"直播加餐｜用OpenAI API理解Python数据类型","content":"","comments":[{"had_liked":false,"id":386469,"user_name":"海蓝的神明","can_delete":false,"product_type":"c3","uid":3797088,"ip_address":"江苏","ucode":"323BE35AB7E643","user_header":"https://static001.geekbang.org/account/avatar/00/39/f0/60/35b88f50.jpg","comment_is_top":false,"comment_ctime":1704882629,"is_pvip":false,"replies":[{"id":140983,"content":"从 这三个方面去找原因：\n1 可能是网络连接不稳定或无法访问服务器\n2 错误的 SSL 版本\n3 如果连接openai ，需要梯子，无法直接连接","user_name":"作者回复","user_name_real":"编辑","uid":1056235,"ctime":1705419002,"ip_address":"广东","comment_id":386469,"utype":1}],"discussion_count":1,"race_medal":0,"score":2,"product_id":100310001,"comment_content":"代码报了一下错误：\n---------------------------------------------------------------------------\nSSLEOFError                               Traceback (most recent call last)\nFile d:\\Python3.10.4\\lib\\site-packages\\httpcore\\_exceptions.py:10, in map_exceptions(map)\n      9 try:\n---&gt; 10     yield\n     11 except Exception as exc:  # noqa: PIE786\n\nFile d:\\Python3.10.4\\lib\\site-packages\\httpcore\\_backends\\sync.py:168, in SyncStream.start_tls(self, ssl_context, server_hostname, timeout)\n    167         self.close()\n--&gt; 168         raise exc\n    169 return SyncStream(sock)\n\nFile d:\\Python3.10.4\\lib\\site-packages\\httpcore\\_backends\\sync.py:163, in SyncStream.start_tls(self, ssl_context, server_hostname, timeout)\n    162         self._sock.settimeout(timeout)\n--&gt; 163         sock = ssl_context.wrap_socket(\n    164             self._sock, server_hostname=server_hostname\n    165         )\n    166 except Exception as exc:  # pragma: nocover\n\nFile d:\\Python3.10.4\\lib\\ssl.py:512, in SSLContext.wrap_socket(self, sock, server_side, do_handshake_on_connect, suppress_ragged_eofs, server_hostname, session)\n    506 def wrap_socket(self, sock, server_side=False,\n    507                 do_handshake_on_connect=True,\n    508                 suppress_ragged_eofs=True,\n    509                 server_hostname=None, session=None):\n    510     # SSLSocket class handles server_hostname encoding before it calls\n...\n    908     &#39;HTTP Request: %s %s &quot;%i %s&quot;&#39;, request.method, request.url, response.status_code, response.reason_phrase\n    909 )\n    911 try:\n\nAPIConnectionError: Connection error.\nOutput is truncated. View as a scrollable element or open in a text editor. Adjust cell output settings...","like_count":0,"discussions":[{"author":{"id":1056235,"avatar":"https://static001.geekbang.org/account/avatar/00/10/1d/eb/b2123759.jpg","nickname":"尹会生","note":"","ucode":"D1093DBD093617","race_medal":0,"user_type":2,"is_pvip":false},"reply_author":{"id":0,"avatar":"","nickname":"","note":"","ucode":"","race_medal":0,"user_type":1,"is_pvip":false},"discussion":{"id":635978,"discussion_content":"从 这三个方面去找原因：\n1 可能是网络连接不稳定或无法访问服务器\n2 错误的 SSL 版本\n3 如果连接openai ，需要梯子，无法直接连接","likes_number":0,"is_delete":false,"is_hidden":false,"ctime":1705419002,"is_liked":false,"can_delete":false,"is_complain":false,"is_top":true,"parent_id":0,"ip_address":"广东","group_id":0},"score":2,"extra":"{\"reply\":true,\"user_type\":2,\"source\":0}","child_discussion_number":0,"child_discussions":[]}]}]}